src/styles/main.scss

```
@@ -0,0 +1,101 @@
+@import "./utils/variables";
+
+.body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  min-height: 100vh;
+
+  font-family: "Audiowide", cursive;
+  background-color: #000;
+}
+
+.stop-watch {
+  min-width: $sw-size;
+  min-height: $sw-size;
+
+  border-radius: 50%;
+  background: radial-gradient(#000 #{$sw-size / 3}, #e76033);
+
+  &__dial {
+    position: relative;
+  }
+
+  &__dial-number {
+    position: absolute;
+    transform: translate(-50%, -50%);
+
+    font-size: $sw-size / 8;
+    text-align: center;
+    text-shadow: 0 0 3px #ff0;
+
+    @each $name, $position in $number-alignment-x {
+      &:nth-child(#{$name}) {
+        top: $position;
+      }
+    }
+
+    @each $name, $position in $number-alignment-y {
+      &:nth-child(#{$name}) {
+        left: $position;
+      }
+    }
+  }
+
+  &__hands {
+    position: relative;
+  }
+
+  &__hand-minutes,
+  &__hand-seconds {
+    position: absolute;
+    left: $sw-size / 2 - $sw-size / 100;
+    bottom: -$sw-size / 2;
+
+    width: $sw-size / 50;
+    height: $sw-size * 0.3;
+
+    border-radius: 40% 40% 0 0;
+    background-color: #87ceeb;
+
+    animation: rotate 3600s linear infinite;
+    transform-origin: 50% 100%;
+  }
+  
+  &__hand-seconds {

```

You cand create class .stop-watch__hand for your hands, and add modifier for second

```suggestion
  &__hand {
    position: absolute;
    left: $sw-size / 2 - $sw-size / 100;
    bottom: -$sw-size / 2;

    width: $sw-size / 50;
    height: $sw-size * 0.3;

    border-radius: 40% 40% 0 0;
    background-color: #87ceeb;

    animation: rotate 3600s linear infinite;
    transform-origin: 50% 100%;
    
    &--seconds {}
  }

```